functions/routes/index.js

```diff
@@ -165,16 +164,41 @@ router.post("/email", async function (req, res, next) {
         next(error);
     }
 });
-
+*/
 router.post("/voicemail", async function (req, res, next) {
-    logger.debug(req.body.phone);
-    logger.debug(req.body.url);
+    const { phone, url, zipcode } = req.body;
+    logger.debug(phone);
+    logger.debug(url);
+    logger.debug(zipcode);
```

Comment:
Instead, you can do like this

```js
const { phone, url, zipcode } = req.body;
logger.debug({phone, url, zipcode});
```

and it will debug all of them with there names as key